shopfloor/tests/test_zone_picking_base.py

```diff
@@ -12,6 +12,24 @@ def setUpClassVars(cls, *args, **kwargs):
         cls.wh = cls.profile.warehouse_id
         cls.picking_type = cls.menu.picking_type_ids
 
+    @classmethod
+    def setUpClassUsers(cls):
+        super().setUpClassUsers()
+        Users = (
+            cls.env["res.users"]
+            .sudo()
+            .with_context({"no_reset_password": True, "mail_create_nosubscribe": True})
+        )
+        cls.stock_user2 = Users.create(
+            {
+                "name": "Paul Posichon",
```

Comment:
:racing_car::racing_car::racing_car:
:joy:

Reply:
For the reference: https://en.wikipedia.org/wiki/Joe_Bar_Team :p